src/blockdata/script.rs

```diff
@@ -325,6 +325,16 @@ impl Script {
                                opcodes::All::from(self.0[0]).classify() == opcodes::Class::IllegalOp)
     }
 
+    /// Gets the minimum value an output with this script should have in order to be
+    /// broadcastable on today's bitcoin network.
+    pub fn dust_value(&self) -> u64 {
+        if self.is_witness_program() {
+            294
```

Comment:
These values could be public constants in case someone needs them without having a script already but knows the context.